### PR TITLE
Fixed genie signature moves

### PR DESCRIPTION
**Pull request by damon-murdoch**
Fixed genie signature moves

## Description
The Genie Signature moves from Legends: Arceus (Bleakwind, Wildbolt & Sandsear Storm) are meant to be spread moves (Both opponents, not partner), but are currently single-target moves. They are also meant to have 100% accuracy in rain. This request makes both of the above mentioned changes.

## Feature(s) this PR does NOT handle:
I have not updated the animations for any of the above moves, so it will still appear to target the leftmost opponent (or whichever target it connects with, if one of the hits miss / are blocked by protect / etc.) Aside from this, the moves work as expected.

## **Discord contact info**
SirScrubbington#0322

### CODE REVIEW

File: src/battle_script_commands.c [hunk 1]
```diff
@@ -1586,11 +1586,14 @@ static bool32 AccuracyCalcHelper(u16 move)
...
     if (WEATHER_HAS_EFFECT)
     {
-        if ((IsBattlerWeatherAffected(gBattlerTarget, B_WEATHER_RAIN) && (gBattleMoves[move].effect == EFFECT_THUNDER || gBattleMoves[move].effect == EFFECT_HURRICANE)))
-        {
-            // thunder/hurricane ignore acc checks in rain unless target is holding utility umbrella
-            JumpIfMoveFailed(7, move);
-            return TRUE;
+        if (IsBattlerWeatherAffected(gBattlerTarget, B_WEATHER_RAIN)){
+            if ((gBattleMoves[move].effect == EFFECT_THUNDER || gBattleMoves[move].effect == EFFECT_HURRICANE) || 
+                (move == MOVE_BLEAKWIND_STORM || move == MOVE_WILDBOLT_STORM || move == MOVE_SANDSEAR_STORM))
```
Comment (AlexOn1ine) [inline src/battle_script_commands.c:1591]: The current code style in the expansion (outside of tests) are brackets on a new line. Also it should suffice to append all the storm moves after Hurricane. I don't think a new `if` is needed in this case.

Comment (damon-murdoch) [reply src/battle_script_commands.c:1591]: New 'if' is not required, I mostly did it this way for readability but if it goes against coding styles / performance issues I'm happy for it to be changed

Comment (AlexOn1ine) [reply src/battle_script_commands.c:1591]: You could move all the checks after Rain to a new line. That would improve readability a bit. Though to be fair there is not a lot of consistency when it comes to long if statements in the repo but usually additional `if` for readability aren't used.

Comment (damon-murdoch) [reply src/battle_script_commands.c]: No worries, cheers for that - I've made a new commit which should have the correct formatting

Comment (AlexOn1ine) [reply src/battle_script_commands.c]: thx

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**